### PR TITLE
Enable fill | write in dir. 

### PR DESCRIPTION
**Pull request by Andrewmatilde**
If path input of disk attack is a dir, we will create a tmp file in the dir & then exec the action.

### CODE REVIEW

**Comment by Andrewmatilde** [comment]
PTAL @cwen0 @WangXiangUSTC .

**Review by cwen0 — APPROVED** [review]
LGTM

File: pkg/core/disk.go [hunk 1]
```diff
@@ -179,35 +179,40 @@ func initPath(opt *DiskOption) (string, error) {
...
+				}
+				return opt.Path, nil
 			}
+			return "", err
 		}
-		return opt.Path, nil
+		if fi.IsDir() {
+			opt.Path, err = utils.CreateTempFile(opt.Path)
+			if err != nil {
+				log.Error(fmt.Sprintf("unexpected err when CreateTempFile in action %s with path %s.", opt.Action, opt.Path))
```
Comment (WangXiangUSTC) [inline pkg/core/disk.go:207]: how about adding error information in the log?

**Review by WangXiangUSTC — COMMENTED** [review]
rest LGTM

File: pkg/core/disk.go (hunk 1 above)
Comment (Andrewmatilde) [reply pkg/core/disk.go:207]: OK,I agree with you.

Comment (Andrewmatilde) [reply pkg/core/disk.go]: Fixed.

**Comment by WangXiangUSTC** [comment]
LGTM

**Review by WangXiangUSTC — APPROVED** [review]

**Pull request merged**